### ISSUE TITLE
Using Json and output interleaves output formats

### ISSUE THREAD

**Issue by HeroicKatora** [JakWai01/lurk#3]
## Reproduction

```
lurk --json -o /tmp/test ls
```

## Expected

The output file `/tmp/test` to represent valid json.

## Actual

```
$ jq < /tmp/test 
[
  7008
]
parse error: Invalid numeric literal at line 1, column 15
```

This is because the file starts with some non-json print *in addition to* the json document:

```
[7008] execve("", "", "") = 0
{"args":["\"\"","\"\"","\"\""],"pid":"7008","result":"0","syscall":"execve","type":"SYSCALL"}[7008] brk(NULL) = 0x55578000
```

**Comment by HeroicKatora** [comment]
But also: thanks, this tool is a fantastic idea and for responding to the biggest interoperability pain point of `strace`.

**Comment by JakWai01** [comment]
Thanks for pointing out this issue. It should be fixed in `v0.2.9`.
